assets/scripts/script.js

```diff
@@ -0,0 +1,85 @@
+let canvas = document.querySelector(".canvas");
+let ctx = canvas.getContext('2d');
+
+ctx.fillStyle = 'rgba(128, 128, 128, 0.5)';
+ctx.fillRect(0, 0, canvas.width, canvas.height);
+
+
+
+let sizeOptionsButtons = document.querySelectorAll('.size-options__button');
+let sizeOptionsButtonImages = document.querySelectorAll('.size-options__button-img');
+for (let i = 0; i < sizeOptionsButtons.length; i++) {
+  sizeOptionsButtons[i].addEventListener('click', function() {
+    for (let j = 0; j < sizeOptionsButtonImages.length; j++) {
+      sizeOptionsButtonImages[j].src = './assets/images/Ellipse.svg';
```

Comment:
And what the reason to put a `for (let j = 0; j < sizeOptionsButtonImages.length; j++) {` inside a `for (let i = 0; i < sizeOptionsButtons.length; i++) {` ?

Reply:
Ammm..., well, it's a complicated way to change button style and neighbour buttons style when we click on one of them. But actually you are right: it would be better to just use classes instead. Agree.